gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb

```diff
@@ -1,842 +1,803 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    for _, (model_name, model) in tqdm(
</pre><p>remove nested tqdm</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Done.